evennia/contrib/grid/extended_room/extended_room.py

```diff
@@ -359,7 +359,10 @@ def func(self):
                 ):
                     detail = location.return_detail(args)
                     if detail:
-                        # we found a detail instead. Show that.
+                        # we found a detail
+                        # tell all the objects in the room we're looking closely at something
+                        caller.location.msg_contents("$You() $conj(look) closely at the $obj(target).\n", from_obj=caller, mapping={"target": args})
```

Comment:
I don't understand why there is `$obj(target)` here, with a mapping of `target=args`. For this code to run, `detail` must be non-Null, which only happens if `args` pointed to a detail (an Attribute on the room). So `$obj()` should never find a result to use here - we should be able to inject `{detail}` directly into the string instead. 

Also, are we sure that `the` is always a suitable prefix to the detail. I would suggest removing `the` from the string.

Reply:
Happy to make the first change, that makes a lot of sense, thankyou.

On the 2nd, the 'the' makes sense when looking at all non-named objects I could think of.  
You look closely at the bed.
Antrare looks closely at the bed. 

It doesn't make sense when looking at named details:
You look closely at the Smaug.
Antrare looks closely at the Smaug.

I don't really mind either way, but it seems like 'the' is going to be right more often than not, and named details are far more likely to be a created object.

Reply:
More flexible is, I think, to skip the prefix: 

You look closely at window 
You look closely at Smaug
You look closely at bed 

These all work in the MUD context, especially since you are likely to write "look bed". 

The dev can create the detail as "the bed" if they want that particular grammatical flavor.

Reply:
Done, removed the 'the'.  

After checking, I believe the $obj(target) /etc. is correct.  The {detail} is actually the full detail description, not the 'key' of the detail.

Reply:
Fair, the `{detail}` is incorrect. It should be `{args}`. My main point is that it makes no sense to wrap it in `$obj()`.